style.css

```diff
@@ -53,11 +53,14 @@
 
   article {
```

Comment:
Hi @SaveryIV 👋

Nice coding and excellent stylesheet.

While you did very nice work here, I noticed that you used a tag selector which is made better by using the class selector. Using a class selector will give you more flexibility and power.

Please consider this change and lovely coding.

Thank you😎